app/file.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+# utilities to check for file existence and accessibility
+def file_exists(name):
+    return os.path.isfile(name)
```

Comment:
Just realized a bug -- this will return False if `name` exists as a directory...